web/styles/drafts.css

```diff
@@ -91,6 +138,11 @@
                     opacity: 1;
                 }
             }
+
+            .checkbox {
```

Comment:
Rather than the fiddly `margin` values here, I wonder if you couldn't--as a separate commit--work up the `message_top_line` div to be a flexbox? Right now, these buttons rely on `inline-block`, which means that it's actual space characters in the HTML that keep them separate from each other. With a flexbox, there'd be more robust control over this bit of microlayout--which seems more important now that there are three controls here instead of two.